metamorph/build.gradle

```diff
@@ -25,6 +25,8 @@ dependencies {
   implementation project(':metafacture-io')
   implementation project(':metafacture-mangling')
   implementation project(':metafacture-javaintegration')
+  implementation "com.google.guava:guava:${versions.guava}"
+  implementation "org.slf4j:slf4j-api:${versions.slf4j}"
   implementation "org.slf4j:slf4j-api:${versions.slf4j}"
```

Comment:
Duplicate dependency specification.

Reply:
huh. Fixed in [f974041](https://github.com/metafacture/metafacture-core/pull/490/commits/f974041665ba41dd1263807e6951c89973d7fc73).